tldw_Server_API/app/api/v1/endpoints/audio.py

```
@@ -1495,15 +1599,25 @@ async def streaming_status():
+        # Check for standard variant (NeMo)
         try:
             from tldw_Server_API.app.core.Ingestion_Media_Processing.Audio.Audio_Transcription_Nemo import (
                 load_parakeet_model
             )
             available_models.append("parakeet-standard")
         except ImportError:
             pass
+
+        # Check for ONNX variant
+        try:
+            import onnxruntime  # noqa: F401
```

Ruff (0.14.2)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)